### PR TITLE
add content about duplicated values in non-transactional-dml.md

### PR DESCRIPTION
**Pull request by ekexium**
Signed-off-by: ekexium <ekexium@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by you06 — APPROVED** [review]

File: non-transactional-dml.md [hunk 1]
```diff
@@ -260,6 +261,14 @@ batch-dml 是一种在 DML 语句执行期间将一个事务拆成多个事务
 
 ## 常见问题
 
+### 分批的大小和指定的 batch size 不一样
```
Comment (TomShawn) [inline non-transactional-dml.md:264]:
```suggestion
### 实际的 batch 大小和指定的 batch size 不一样
```

File: non-transactional-dml.md [hunk 2]
```diff
@@ -260,6 +261,14 @@ batch-dml 是一种在 DML 语句执行期间将一个事务拆成多个事务
...
+
+最后一个 batch 处理的数据可能会少于 batch size。
```
Comment (TomShawn) [inline non-transactional-dml.md:266]:
```suggestion
在非事务 DML 语句的执行过程中，最后一个 batch 处理的数据量可能会小于 batch size。
```

File: non-transactional-dml.md [hunk 3]
```diff
@@ -260,6 +261,14 @@ batch-dml 是一种在 DML 语句执行期间将一个事务拆成多个事务
...
+
+在**划分列有重复值**时，每个 batch 会将最后一个元素的所有重复值全部加入当前这个 batch 中，因此这个 batch 处理的行数可能会多于 batch size。
```
Comment (TomShawn) [inline non-transactional-dml.md:268]: 这里的「元素」是指行吗？

Comment (ekexium) [reply non-transactional-dml.md:268]: 是指划分列 $C$ 的值

Comment (TomShawn) [reply non-transactional-dml.md:268]:
```suggestion
在**划分列有重复值**时，每个 batch 会将划分列最后一个元素的所有重复值全部加入最后一个 batch 中，因此最后一个 batch 处理的行数可能会多于 batch size。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**